File: Labs/lab03/ip.py
ip = "10.0.0.1"
parts = ip.split('.')
nums = [int(x) for x in parts]

nums[0] = nums[0] << 24
nums[1] = nums[1] << 16
nums[2] = nums[2] << 8
nums[3] = nums[3]

print(sum(nums))

File: Labs/lab03/ip.js
function ipToNumber(ip) {
    const parts = ip.split('.').map(x => parseInt(x));
    
    const nums = [
        parts[0] << 24,
        parts[1] << 16,
        parts[2] << 8,
        parts[3]
    ];

    const total = nums.reduce((x, y) => x + y, 0);
    return total;
}

console.log(ipToNumber('10.0.0.1'));

console.log(ipToNumber('127.0.0.1'));

console.log(ipToNumber('192.168.1.10')); 